services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None
+) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+    return MovieSession.objects.all()
```

Comment:
You can store all `MovieSession` in a variable and if the condition in line 23 is `True` -> apply needed filters. In such approach you can avoid using two `returns`